qa/vagrant/src/test/resources/packaging/tests/60_systemd.bats

```diff
@@ -236,3 +236,12 @@ setup() {
     [ "$max_address_space" == "unlimited" ]
     systemctl stop elasticsearch.service
 }
+
+@test "[SYSTEMD] test runtime directory" {
+    clean_before_test
+    install_package
+    sudo rm -rf /var/run/elasticsearch
+    systemctl start elasticsearch.service
+    wait_for_elasticsearch_status
```

Comment:
Thanks for the suggestion; I pushed a check in 7c83a2f3f1e72ea5448a36bbd46d6ce49585c68e.